hail/src/main/scala/is/hail/io/IndexBTree.scala

```diff
@@ -204,7 +203,7 @@ class IndexBTree(indexFileName: String, hConf: Configuration, branchingFactor: I
 // like an on-disk array and looks up values by index
 class OnDiskBTreeIndexToValue(
   path: String,
-  hConf: Configuration,
+  sFS: FS,
```

Comment:
Maybe should be fs; there's some ambiguity here because fs is also used to refer to the FileSystem object returned from FS.fileSystem. We can choose a different name globally if you prefer as well.

Reply:
changing to fs: FS, and fileSystem for instances of FileSystem